src/main/java/org/folio/rest/helper/FiscalYearsHelper.java

```diff
@@ -28,8 +33,15 @@ public FiscalYearsHelper(HttpClientInterface httpClient, Map<String, String> oka
     super(httpClient, okapiHeaders, ctx, lang);
   }
 
-  public CompletableFuture<FiscalYear> createFiscalYear(FiscalYear fund) {
-    return handleCreateRequest(resourcesPath(FISCAL_YEARS), fund).thenApply(fund::withId);
+  public CompletableFuture<FiscalYear> createFiscalYear(FiscalYear fiscalYear) {
+    return HelperUtils.getConfigurationEntries(okapiHeaders, ctx, logger)
+      .thenCompose(locale -> {
+        // Initially the config will not have any locale, the same default values used here are hardcoded in tenant Settings
+        fiscalYear.setCurrency(new JsonObject(
+            locale.getString(LOCALE_SETTINGS, DEFAULT_LOCALE))
```

Comment:
It is theoretically possible that the configuration will have `LOCALE_SETTINGS`, but there will be no currency in it.

Reply:
added additional tests